build.gradle.kts

```diff
@@ -17,6 +17,7 @@ buildscript {
 
 plugins {
     id("com.osacky.doctor")
+    id("com.gradleup.auto.manifest") version "1.0.3" apply false
```

Comment:
Please let me know how to add this version into `versions.properties` and I will do it.

Reply:
You can add `plugin.thepluginid` so `plugin.com.gradleup.auto.manifest=1.0.3`.
You can see how I do it for Gradle doctor already.